### ISSUE TITLE
torch.cat( (torch.max(x, 1)[0].unsqueeze(1), torch.mean(x, 1).unsqueeze(1)), dim=1 )

### ISSUE THREAD

**Issue by Niki173** [LandskapeAI/triplet-attention#19]
triplet-attention.py

Hi,Run this file and find inconsistent dimensions

I wonder if there is a problem here

torch.cat( (torch.max(x, 1)[0].unsqueeze(1), torch.mean(x, 1).unsqueeze(1)), dim=1 )

it should be  torch.cat( (torch.max(x, 1).unsqueeze(1), torch.mean(x, 1).unsqueeze(1)), dim=1 )

right?

**Comment by digantamisra98** [comment]
@Niki173 apologies for the delay in response, for Z-pool, we follow the same code as that given in [CBAM](https://github.com/Jongchan/attention-module/blob/5d3a54af0f6688bedca3f179593dff8da63e8274/MODELS/cbam.py#L68)
